docinfo-footer.html

```diff
@@ -1,3 +1,49 @@
 <div id="footer-github">
     Edit these docs <a href="https://github.com/LibrePCB/librepcb-doc/">on GitHub</a>!
 </div>
+<script type="text/javascript">
+  function tocLinkCloseAll() {
+    Array.from(document.querySelectorAll(".toc-visible")).forEach(function(item) {
+      item.classList.remove("toc-visible");
+      item.classList.add("toc-invisible");
+    });
+  }
+  function tocLinkOpen(ul) {
+    ul.classList.remove("toc-invisible");
+    ul.classList.add("toc-visible");
+  }
+  function tocLinkOpenAncestors(ul) {
+    var ancestor = ul.parentNode.closest("ul");
```

Comment:
I'd use ES2015 const-bindings:

```suggestion
    const ancestor = ul.parentNode.closest("ul");
```

Same thing for other `var` bindings.

Reply:
Same thing for my lack of ES2015 skills... 😆